week9/algoorgoal/src/apis/cart.ts

```diff
@@ -0,0 +1,28 @@
+import { MenuItemType, MenuType, StoreType } from '../models/stores';
+
+export const getCart = async () => {
+  const response = await fetch('http://localhost:8080/cart');
+  const data = await response.json();
+  return data;
+};
```

Comment:
기왕 typescript 사용하셨다면 Cart에 대한 interface 생성하고 getCart의 반환 타입을 지정하면 어떨까요

```suggestion
export interface ICart {
  id: number;
  menus: Menu[];
}

export const getCart = async (): Cart => {
  const response = await fetch('http://localhost:8080/cart');
  const data: Cart = await response.json();
  return data;
};
```